lua/ulx/modules/sh/cfc_curse.lua

```diff
@@ -40,6 +70,17 @@ function cmd.cursePlayers( callingPlayer, targetPlayers, effectName, durationMin
         type( effectName ) == "string" and
         effectName ~= "random" and
         ( not shouldUncurse or effectName ~= "all" )
+    local amount = isSpecificEffect and not shouldUncurse and tonumber( effectName ) or nil
+
+    if amount == 1 then
+        amount = false
+        isSpecificEffect = false
+        effectName = "random"
+    end
+
+    if amount then
+        isSpecificEffect = false
+    end
```

Comment:
Find the amount to multi-curse by, and handle edge-cases like making `ulx curse target 1` equivalent to `ulx curse target random`, not doing multi-curse when uncursing, and preventing the command from trying to interpret the amount as a curse name.